app/scripts/CenterTrack.jsx

```diff
@@ -1,4 +1,3 @@
-// @ts-check
```

Comment:
I'd used this previously for my own sanity when refactoring, but these files currently won't pass the type checker.